src/locale/messages.fi.xlf

```diff
@@ -1,345 +1,365 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-  <file source-language="en" datatype="plaintext" original="ng2.template" target-language="fi">
+  <file source-language="en" datatype="plaintext" original="ng2.template" target-language="fi-FI">
```

Comment:
When building the GUI, I get the following error :
```
WARNING [src/locale/messages.fi.xlf]: File target locale ('fi-FI') does not match configured locale ('fi')
```
I think this is related to this update. Is this something we should consider reverting?

Reply:
It looks like it is related to this update. All other xlf files seems to have just "xx". I will test and then create a PR that fixes it (chanes "fi-FI" to "fi").